projects/magicsea-lb-v2/index.js

```diff
@@ -0,0 +1,37 @@
+const {sumTokens2} = require('../helper/unwrapLPs')
+
+const factories = {
+    shimmer_evm: '0xEE0616a2DEAa5331e2047Bc61E0b588195A49cEa',
```

Comment:
noticed that we already count this as part of swapline, so excluded it

Reply:
Hi @g1nt0ki MagicSea and Swapline are from the same team. After your comment we had a discussion in our team and we agreed to have the LB metrics for Shimmer on Swapline removed in favour to show them in MagicSea. 
Could you please take care of this? Or do you prefer that I create one PR in Swapline and one PR in MagicSea?

Reply:
Another question is that MagicSea and MagicSea LB are currently treated as two separate projects. Unlike e.g. TraderJoe where the LB part is a subproject of TraderJoe. 
Can you give us advice what we need to do to have MagicSea LB visible in a tab within the "main" MagicSea DEX?

Reply:
https://defillama.com/protocol/magicsea here is the parent protocol with the both the sub protocols :)